sumpy/qbx.py

```diff
@@ -183,29 +183,30 @@ def get_kernel(self):
         raise NotImplementedError
 
     def get_optimized_kernel(self,
-            targets_is_obj_array, sources_is_obj_array, centers_is_obj_array):
+            targets_is_obj_array, sources_is_obj_array, centers_is_obj_array,
+            itgt_name="itgt"):
```

Comment:
For my education: where does `itgt` need to be overridden?

Reply:
It's used at https://github.com/inducer/pytential/pull/75/files#diff-4fcf38e9154a831bf71a50806d418d2ce5fbdd57cdf44c979ffc90f919e03b1dR123-R127

Reply:
```suggestion
            # Used by pytential to override the name of the loop to be
            # parallelized. In the case of QBX, that's the loop over QBX
            # targets (not global targets).
            itgt_name="itgt"):
```

Reply:
FWIW, it took me about 20 minutes of puzzling to convince myself that what you're doing here is correct, and I'm not sold the rename makes things clearer... to be fair, I'm not claiming that the code was clear or beautiful to start with.